src/sarif-converter-21.ts

```diff
@@ -26,6 +29,9 @@ export function defaultSarifConverter21(): SarifConverter21 {
     );
 }
 export class SarifConverter21 {
+    private readonly wcagLinkDataProcessor: WCAGLinkDataProcessor = new WCAGLinkDataProcessor(
```

Comment:
The word "Processor" in this variable name/class name confused me a little; I wasn't sure what the point of this class was intended to be until I saw the usage/implementation later. Maybe "Index" or "Indexer" would be more clear?